content/docs/user-guide/assets/pipeline/scan-directories.md

```diff
@@ -20,7 +20,40 @@ Asset processor monitors the following scan directories:
+
+## Files in scan directories
+
+Scan directories provide the entry point for [asset builders](docs/user-guide/assets/pipeline/asset-builders/) to read files as source assets, for generating product assets into the asset cache.
+
+A file being in a scan directory does not automatically become a source asset. Source assets are files in scan directories that have had jobs created for them. Otherwise, these files are called non-asset source files. A non-asset source file will not have a UUID to identify it, and can only be referenced by path.
+
+## Referencing files in scan directories
+
+There are three core points during development where it may be necessary to reference and potentially load a file that is in a scan directory.
+
+In all cases, it's important to make sure [dependencies](docs/user-guide/assets/pipeline/asset-dependencies-and-identifiers/) are accurate for all assets.
+
+### From asset builders
+
+When authoring [asset builders,](docs/user-guide/assets/pipeline/asset-builders/) loading the primary source asset used by the builder is straightforward. The create jobs and process job functions each have a parameter with the path to the relevant source asset.
+
+Sometimes it may be necessary to load additional files besides the primary source asset within these builders. See [this documentation on loading other files](docs/user-guide/assets/pipeline/asset-builders/#loading-other-files) for details on handling these situations.
+
+### From external tools
+
+In many cases, the files intended to be source assets that will be placed into scan folders, are authored by tools external to O3DE. In some cases, these may be standard, portable file formats generated by many different external tools, such as FBX files. In other cases, these may be files generated only by a specific tool. In either case, the asset builder authors may not have the ability to modify the source file format.
+
+When building out the pipeline, ideally the source asset file format can be designed to avoid relative paths or absolute paths to other files.
+
+When this can't be done, such as with an existing file format like FBX files referencing textures in materials, then these paths will need to be resolved in some fashion during the asset builder process. Sometimes these paths will be absolute paths from the machine that generated the source asset. Sometimes they will be relative paths that only resolve correctly on the machine that generated the source asset. How to handle this will often come down to specifics about the asset builder being authored, however the most common solution is to ignore all of the path to the referenced asset except the filename and extension, and then search for a file matching that in the same directory as the asset being processed, dropping the reference entirely if the referenced file isn't found there.
```

Comment:
```suggestion
When this can't be done, such as with an existing file format like FBX files referencing textures in materials, then these paths will need to be resolved in some fashion during the asset builder process. Sometimes these paths will be absolute paths from the machine that generated the source asset. Sometimes they will be relative paths that only resolve correctly on the machine that generated the source asset. How to handle this will often come down to specifics about the asset builder being authored, however the most common solution is to ignore all of the path information to the referenced asset except the filename and extension, and then search for a file matching that in the same directory as the asset being processed, dropping the reference entirely if the referenced file isn't found there.
```